feat: reverse proxy our own endpoints

raquelmsmith opened the pull request:
## Changes

Our capture and feature flag endpoints can be blocked by adblockers. We can reverse proxy our own endpoints so they aren't blocked.

smallbrownbike commented:
Vercel actually uses `.env.production` for preview builds! And the billing variable will break builds for peeps that don’t have a local billing server running so I reverted the `.env.development` change.

raquelmsmith commented:
> Vercel actually uses .env.production for preview builds!

Hm, it didn't seem to do that, since I changed `.env.production` and the preview build didn't use those vars - it only worked after I updated the vercel vars.

smallbrownbike commented:
You're right!! Reverted part 2 sans the billing variable
